content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -323,6 +323,53 @@ the files in the following order:
 1. `support/component.js` (your support file)
 2. `components/Button/Button.cy.js` (your spec file)
 
+#### Use supportFile to load scripts before your test code
+
+It's still useful to load a setup file before your test code. If you are setting
+Cypress defaults or utilizing custom Cypress commands, instead of needing to
+import/require those defaults/commands in every test file, you can use the
+[`supportFile`](/guides/references/configuration#Folders-Files) configuration
+option.
+
+To include code before your test files, set the
+[`supportFile`](/guides/references/configuration#Folders-Files) path. By
+default, [`supportFile`](/guides/references/configuration#Folders-Files) is set
+to look for one of the following files:
+
+**Component:**
```

Comment:
I added a couple of variations here that we discussed in comments. That was before some other incoming changes from another PR. 

We can go with the first text block in this section or one of these two formatting options.